### PR TITLE
fix(client) new blog component and homepage

### PR DESCRIPTION
**Pull request by nihalanisumit**
## PR Checklist
Please check that your PR fulfills the following requirements:

- [X] The commit message follows [our guidelines](https://github.com/pesto-students/batch-9-Lumen/wiki/Commit-message-convention)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [X] GIFs/Videos/Images added (only for frontend changes)

## PR Type
What kind of change does this PR introduce? 

- [X] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Infrastructure changes
- [ ] Other... Please describe:
Issue Number: 
#155 #156 

## Does this PR introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by nihalanisumit** [comment]
newblogcard gif

File: client/src/containers/Home/Home.jsx [hunk 1]
```diff
@@ -2,70 +2,70 @@
...
+
   return (
     <div className={styles.background}>
       <h1 className={styles.title}> Top trending blogs</h1>
       <div className={styles.container}>
         {fetched ? (
           <>
             <div className={[styles.column, styles['column-first']].join(' ')}>
-              {biggerCards}
+              {/* {biggerCards} */}
```
Comment (nihalanisumit) [inline client/src/containers/Home/Home.jsx:44]: will uncomment when I will be fixing home page css.

File: client/src/containers/Home/Home.jsx [hunk 2]
```diff
@@ -2,70 +2,70 @@
...
   const smallerCardsFirstColumn = blogs
-    .slice(2, 6)
+    .slice(4, 8)
     .map(blog => getBlog(blog, 'small-blog'));
   const smallerCardsSecondColumn = blogs
-    .slice(6)
+    .slice(8,12)
     .map(blog => getBlog(blog, 'small-blog'));
 
+  const sampleBlogCard = () => <BlogCard />
```
Comment (piyushagwl007) [inline client/src/containers/Home/Home.jsx:35]: Can you shuffle the array randomly if possible? It might be fun to see blogs getting random order?

**Review by piyushagwl007 — APPROVED** [review]
LGTM

**Comment by piyushagwl007** [comment]
Great work :smile:

**Pull request merged**